tests/testDisplay.py

```diff
@@ -172,35 +172,15 @@ def testInteract(self):
         """
         self.display0.interact()
 
-#-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
-
-
-def suite():
-    """Returns a suite containing all the test cases in this module."""
-    tests.init()
 
-    suites = []
-    suites += unittest.makeSuite(DisplayTestCase)
-    suites += unittest.makeSuite(tests.MemoryTestCase)
-    return unittest.TestSuite(suites)
+class MemoryTester(lsst.utils.tests.MemoryTestCase):
+    pass
 
 
-def run(shouldExit=False):
-    """Run the tests"""
+def setup_module(module):
+    lsst.utils.tests.init()
 
-    tests.run(suite(), shouldExit)
 
 if __name__ == "__main__":
-    import argparse
```

Comment:
Please add this argument parsing code back in. It doesn't hurt anything and we have no reason to display the ability to specify a display backend on the command line.